azure-communication-ui/calling/src/androidTest/java/com/azure/android/communication/ui/calling/LobbyTest.kt

```diff
@@ -397,7 +399,9 @@ internal class LobbyTest : BaseUiTest() {
 
     private fun joinTeamsCall(): Context? {
         val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        val callComposite = CallCompositeBuilder().build()
+        val telecomOptions =

```

Comment:
What, if not provided is the default? Can there be a TelecomOptions non-nullable instance that serves as the default if not assigned?